spacegraphcats/snakemake.py

```diff
@@ -5,29 +5,31 @@
 def catlas_build(conf_file):
     "Produce the list of files output by 'spacegraphcats <config> build"
     with open(conf_file, "rt") as fp:
-        jj = yaml.load(fp)
+        jj = yaml.safe_load(fp)
 
     catlas_base = jj["catlas_base"]
     ksize = jj["ksize"]
     radius = jj["radius"]
 
-    dirname = "{}_k{}_r{}".format(catlas_base, ksize, radius)
+    cdbg_dir = f"{catlas_base}_k{ksize}"
+    catlas_dir = f"{catlas_base}_k{ksize}_r{radius}"
 
     z = []
-    z.append(os.path.join(catlas_base, "bcalm.{}.k31.unitigs.fa".format(catlas_base)))
-    z.append(os.path.join(dirname, "catlas.csv"))
-    z.append(os.path.join(dirname, "cdbg.gxt"))
-    z.append(os.path.join(dirname, "contigs.fa.gz.indices"))
-    z.append(os.path.join(dirname, "contigs.fa.gz.info.csv"))
-    z.append(os.path.join(dirname, "contigs.fa.gz.mphf"))
-    z.append(os.path.join(dirname, "first_doms.txt"))
+    z.append(os.path.join(cdbg_dir, "bcalm.unitigs.db")),
```

Comment:
removed the hardcoded `k31` in the name bc the k size is flexible from the conf file?

Reply:
ooh, good catch!

Reply:
oh. I misunderstood.

Umm, the ksize is already in the directory name here.